sycl/test/on-device/sub_group/info.cpp

```diff
@@ -23,37 +21,33 @@ int main() {
   queue Queue;
   device Device = Queue.get_device();
 
-  /* Basic sub-group functionality is supported as part of cl_khr_subgroups
-   * extension or as core OpenCL 2.1 feature. */
-  if (!core_sg_supported(Device)) {
```

Comment:
Why do you remove the check? There might be a case that OLD CPU/GPU RT is used which does not support subgroups.